helpers/utils.py

```diff
@@ -0,0 +1,291 @@
+from pymongo import (MongoClient, UpdateOne,)
+import psycopg2
+from pandas.io.json._normalize import nested_to_record
+
+from .configs import (
+    POSTGRES_DB, POSTGRES_DB_PASSWORD, POSTGRES_USER,
+    POSTGRES_HOST, POSTGRES_PORT, SURV_MONGO_URI,
+)
+
+
+############################
+#  Database Connections    #
+############################
+def establish_mongo_connection(db_name):
+    """
+    establish MongoDB connection
+    : param mongo_uri: mongo connection uri
+    : return db_connection: database connection
+    """
+    client = MongoClient(SURV_MONGO_URI)
+
+    db_connection = client[db_name]
+    return db_connection
+
+
+def establish_postgres_connection():
+    """
+    establish postgres connection
+    :return connection: database connection
+    """
+    connection = psycopg2.connect(
```

Comment:
Hi @odenypeter, is there a reason to use the older `'{}'.format()` rather than f-strings (`f"{}"`)?

Reply:
@shinroo backword compatibility is something that I tried to put first. The moment I do f-strings, then, guys with old airflow dags based on python2.7 or early 3.0 will have problems using our dags out of the box.